hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/RouterServerUtil.java

```diff
@@ -227,7 +227,7 @@ public static void logAndThrowIOException(String errMsg, Throwable t)
       throws IOException {
     if (t != null) {
       LOG.error(errMsg, t);
-      throw new IOException(errMsg, t);
```

Comment:
We made changes to this part of the code because we aim to convey more information to the client.